.github/workflows/benchmarking.yml

```diff
@@ -329,9 +299,9 @@ jobs:
       SAVE_PERF_REPORT: ${{ github.event.inputs.save_perf_report || ( github.ref == 'refs/heads/main' ) }}
       PLATFORM: ${{ matrix.platform }}
 
-    runs-on: [ self-hosted, dev, x64 ]
+    runs-on: [ self-hosted, us-east-2, x64 ]
     container:
-      image: 369495373322.dkr.ecr.eu-central-1.amazonaws.com/rustlegacy:pinned
```

Comment:
👍 Is that the last usage? We can remove https://github.com/neondatabase/build/tree/main/rustlegacy then.

Reply:
Yep, this was the last one. I'll create a PR with the removal of this image from another repo.